src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayFormat.cs

```diff
@@ -27,14 +27,16 @@ public class SymbolDisplayFormat
                 parameterOptions:
                     SymbolDisplayParameterOptions.IncludeParamsRefOut |
                     SymbolDisplayParameterOptions.IncludeType,
-                // Not showing the name is important because we visit parameters to display their
-                // types.  If we visited their types directly, we wouldn't get ref/out/params.
                 miscellaneousOptions:
                     SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers |
                     SymbolDisplayMiscellaneousOptions.UseSpecialTypes |
                     SymbolDisplayMiscellaneousOptions.UseAsterisksInMultiDimensionalArrays |
                     SymbolDisplayMiscellaneousOptions.UseErrorTypeSymbolName |
-                    SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier);
+                    SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier,
+                compilerInternalOptions: SymbolDisplayCompilerInternalOptions.IncludeParameterNameIfStandalone);
+
+        internal static SymbolDisplayFormat CSharpErrorMessageTypeFormat { get; } = CSharpErrorMessageFormat
```

Comment:
>CSharpErrorMessageTypeFormat [](http://example.com/codeflow?start=44&length=28)

It feels like it would be better to make this format about parameters rather than about types. For example, "CSharpErrorMessageNoParameterNamesFormat".

Reply:
Thanks, no problem, changed: 503f5f7e713191070888ce72555caf911a1d131d